.github/PULL_REQUEST_TEMPLATE/release_template.md

```diff
@@ -1,17 +1,15 @@
-Link to the milestone on Github e.g. https://github.com/nimblehq/git-templates/milestone/41?closed=1
-or
-Link to the project management tool for the release
+https://github.com/nimblehq/flutter-templates/milestone/?
```

Comment:
```suggestion
https://github.com/nimblehq/kmm-templates/milestone/?
```
😂

Reply:
Good catch. Fixed in https://github.com/nimblehq/kmm-templates/pull/11/commits/a567f565ab72361b85c443e6a3849f540a29c2d0